package.json

```diff
@@ -70,45 +62,15 @@
     "@stdlib/number-float64-base-normalize": "0.0.8",
     "@swc/core": "^1.5.7",
     "@types/express": "^4.17.13",
-    "@types/jest": "^27.4.0",
     "@types/mime": "1.3.5",
     "@types/node": "^17.0.8",
-    "babel-preset-medusa-package": "^1.1.19",
     "chokidar": "^3.6.0",
     "cross-env": "^7.0.3",
     "eslint": "^6.8.0",
     "execa": "^9.1.0",
-    "jest": "^27.3.1",
     "rimraf": "^3.0.2",
-    "ts-jest": "^27.0.7",
-    "ts-loader": "^9.2.6",
     "ts-node": "^10.9.2",
     "typescript": "^4.5.2"
   },
-  "jest": {
-    "globals": {
-      "ts-jest": {
-        "tsconfig": "tsconfig.spec.json"
-      }
-    },
-    "moduleFileExtensions": [
-      "js",
-      "json",
-      "ts"
-    ],
-    "testPathIgnorePatterns": [
-      "/node_modules/",
-      "<rootDir>/node_modules/"
-    ],
-    "rootDir": "src",
-    "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|js)$",
-    "transform": {
-      ".ts": "ts-jest"
-    },
-    "collectCoverageFrom": [
-      "**/*.(t|j)s"
-    ],
-    "coverageDirectory": "./coverage",
-    "testEnvironment": "node"
-  }
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
can we remove this?

Reply:
Ohh yeah. Sorry, my corepack stuff leaked into the package.json file